README.md

```diff
@@ -13,6 +13,40 @@ certificate authority. After issuance, the certificate is then returned to Hashi
 
 ---
 
+## Table of contents
```

Comment:
Nice did you have something generate this for you?

Reply:
That would have been nice!  no, I wrote it by hand.